app/components/button_locale/ButtonLocale.tsx

```diff
@@ -0,0 +1,18 @@
+import nextI18nextConfig from "../../../next-i18next.config"
+import LanguagesToogle from "../language_toogle/LanguagesToogle"
+
+interface ButtonLocaleProps {
+    currentLocale: string | string[]
+}
+export default function ButtonLocale({ currentLocale }: ButtonLocaleProps) {
```

Comment:
currently on the project we are setting the export default at the end of the component

Reply:
I understand you, it was resolved